src/asciidoc/docs/_includes/redirects.adoc

```diff
@@ -0,0 +1,20 @@
+= Redirects
```

Comment:
It looks like we’re not using this include page anywhere yet. We could think about applying this note to some of the pages in the lower-priority updates bucket. At our meeting today, let’s go through the roadmap and discuss whether we want to do it.

Reply:
Okies!